src/main/config/feature.h

```diff
@@ -51,6 +51,7 @@ typedef enum {
     FEATURE_ESC_SENSOR = 1 << 27,
     FEATURE_ANTI_GRAVITY = 1 << 28,
     FEATURE_DYNAMIC_FILTER = 1 << 29,
+	FEATURE_LEGACY_SA_SUPPORT = 1 << 30,
```

Comment:
I know its merely cosmetic but can we line it up ?

Reply:
Done, sorry, Visual Studio generates a tab from 3 spaces by default -.-, turned it off.